build/mvn

```diff
@@ -119,7 +119,8 @@ install_mvn() {
   if [ "$MVN_BIN" ]; then
     local MVN_DETECTED_VERSION="$(mvn --version | head -n1 | awk '{print $3}')"
   fi
-  if [ $(version $MVN_DETECTED_VERSION) -lt $(version $MVN_VERSION) ]; then
```

Comment:
why not use exact match here?

Reply:
I am not sure when 3.9.x will be supported due to there is a pre-work need to complete, if the pre-work cannot be completed, the version after 3.9.0 also needs to be temporarily disabled

Reply:
I'm change `-lt` to `-ne`, and always respect `maven.version` defined in `pom.xml`

Reply:
Good idea

Reply:
done